state/state.go

```diff
@@ -152,5 +162,17 @@ func flagsForChart(chart *ChartSpec) ([]string, error) {
 		}
 		flags = append(flags, "--set", strings.Join(val, ","))
 	}
+	if len(chart.EnvValues) > 0 {
+		val := []string{}
+		for _, set := range chart.EnvValues {
+			value, isSet := os.LookupEnv(set.Value)
+			if isSet {
+				val = append(val, fmt.Sprintf("%s=%s", set.Name, value))
+			} else {
+				return nil, errors.New(fmt.Sprintf("Unset env var: %s", set.Name))
```

Comment:
nit: This error message could be read as an instruction, e.g. please unset env var {}. Can you change it to something clearer?